DOCS.md

```diff
@@ -1,9 +1,8 @@
-![true[X] logo](media/truex.png)
-
+![true\[X\] logo](media/truex.png)
 
 # TruexAdRenderer tvOS Documentation
 
-Version 3.6
+Version 3.6.0
```

Comment:
Hmm, may want to keep it generic since we would recommend publishers consume the latest `3.6.x`

Reply:
True